### ISSUE TITLE
/api/message/reset

### ISSUE THREAD

**Issue by ryunalore** [ryunalore/SLM#8]
- Provide function that requires the parameter "reset"
- When executed it resets the "current state" to null
   - example no.1 - state has been change:
       ``` 
           1. execute /api/message/set?m=Service+checks:+No+power+until+5:00+pm
           2. result: "ok"
           3. execute /api/message
           4. "Service checks: No power until 5:00 pm"
           5. execute /api/message/reset
           6. result: "ok"
           7. execute /api/message
           8. result: "Everything works as expected"
       ```
- Realise as PUT Request
- Return HTTP Status Code 200 (successful)

**Comment by ryunalore** [comment]
@ic21b006 @kaplanilhan Code is now implemented  (see commit [6eede86](https://github.com/ryunalore/SLM/commit/6eede86feab265a72314e70a41a2c54fc70bda13) )



![Image](https://user-images.githubusercontent.com/96705308/204165617-a22a608f-b77a-49bd-b898-b798fc110354.png)



![Image](https://user-images.githubusercontent.com/96705308/204165625-b05e4fdc-4514-4264-b05e-9fd0b6b72d75.png)



![Image](https://user-images.githubusercontent.com/96705308/204165631-316bdc20-6116-400d-83d7-83a52b7fd1e1.png)

**Comment by ryunalore** [comment]
changed to @GetMapping
